indy_node/test/request_handlers/conftest.py

```diff
@@ -43,6 +49,60 @@ def schema_request():
                               }})
 
 
+@pytest.fixture(scope="module")
+def rs_schema_handler(db_manager, write_auth_req_validator):
+    return RsSchemaHandler(db_manager, write_auth_req_validator)
+
+
+@pytest.fixture(scope="function")
+def rs_schema_request():
+    authors_did, name, version, type = "2hoqvcwupRTUNkXn6ArYzs", randomString(), "1.1", MARKER_RS_SCHEMA
+    _id = authors_did + ':' + type + ':' + name + ':' + version
+    return Request(identifier=authors_did,
+                   reqId=random.randint(1, 10000000000000000000),
+                   signature="sig",
+                   protocolVersion=2,
+                   operation={
+                       'type': SET_RS_SCHEMA,
+                       RS_META: {
+                           RS_META_ID: _id,
```

Comment:
Are `RS_META_ID` and `RS_META_TYPE` mandatory fields or not?

Reply:
`RS_META_ID` is not mandatory I will remove it. `RS_META_TYPE` is mandatory and should equal `RS_SCHEMA_META_TYPE`, I will correct that.